adapters/saml/core/src/main/resources/schema/keycloak_saml_adapter_1_9.xsd

```diff
@@ -321,6 +321,11 @@
                 <xs:documentation></xs:documentation>
             </xs:annotation>
         </xs:attribute>
+        <xs:attribute name="metadataUrl" type="xs:string" use="optional">
```

Comment:
Please introduce a new version, keycloak_saml_adapter_1_10.xsd, instead of updating the old one.

Reply:
Sure, silly of me to update the existing one which is obviously out in the wild already.